examples/calibration/plot_calibration.py

```diff
@@ -0,0 +1,107 @@
+"""
+======================================
+Probability calibration of classifiers
+======================================
+
+When performing classification you often want to predict, not only
+the class label, but also the associated probability. This probability
+gives you some kind of confidence on the prediction. This example
+demonstrates how to transform the decision function of a generic
+classifier into a calibrated probability. The accuracy is estimated
+with Brier's score (see http://en.wikipedia.org/wiki/Brier_score).
+
```

Comment:
Please add a note here stating that in that case, only the non-parametric isotonic calibration method is able to provide a probability calibration that return probabilities close to the expected 0.5 for most of the samples belonging to the middle cluster with heterogeneous labels.